docs/AGREEMENT.md

```diff
@@ -39,3 +39,17 @@ I promise to provide valuable critique necessary for the improvement of the proj
 I plan to take an active role in learning and contributing to all areas of the project. I will ensure to collaborate with the team on every deliverables required for the goals, sprints and milestone for this project and for the class.
 
 If conflict arises, I will ensure that all perspectives are heard and work on a solution-based resolution.
+
+
+### Juan's Agreement
+I, Juan (Does it have to be this formal?), agree to the following:
+
+1) I hope to make a contributions in coding and working on game design for the application.
+
+2) If a conflict were to arise, I'd like to go over what started said conflict to see what we can do to resolve it. If for some reason it can't be resolved whether, technical or personal,
```

Comment:
```suggestion
2. If a conflict were to arise, I'd like to go over what started said conflict to see what we can do to resolve it. If for some reason it can't be resolved whether, technical or personal,
```